include/mbedtls/error.h

```diff
@@ -52,7 +52,7 @@
  * GCM       2  0x0012-0x0014
  * BLOWFISH  2  0x0016-0x0018
  * THREADING 3  0x001A-0x001E
- * AES       2  0x0020-0x0022
+ * AES       2  0x0020-0x0022   0x0021-0x0021
```

Comment:
I don't think that this addition is necessary, as `0x0021` is within the range `0x0020-0x0022`

Reply:
@RonEld It is necessary because odd values and even values are treated independently. See e.g. MPI vs HMAC_DRBG, and OID.

Reply:
@gilles-peskine-arm Why does it matter that they are treated differently, to the comment on the error ranges?

Reply:
@RonEld Because 0x0020-0x0022 is an even range, so it only includes the values 0x0020 and 0x0022, not 0x0021. The value 0x0021 could be used by a different module. For example MPI claims the even range 0x0002-0x0010, while HMAC_DRBG claims the odd range 0x0003-0x0009.